packages/back-end/src/routers/organizations/organizations.controller.ts

```diff
@@ -96,6 +93,13 @@ import {
   getExperimentsForActivityFeed,
 } from "../../models/ExperimentModel";
 import { removeEnvironmentFromSlackIntegration } from "../../models/SlackIntegrationModel";
+import {
+  findAllByEntityType,
+  findAllByEntityTypeParent,
+  findByEntity,
+  findByEntityParent,
```

Comment:
Can we add the word "audit" in these method names? I imagine if every model defined these generic methods that we would have import collisions and would need to remap. In our other models we use the resource name, e.g. `createProject` or `findProjectById` etc.